server/setup.py

```diff
@@ -1,13 +1,19 @@
+import pathlib
 from setuptools import find_packages, setup
 
+here = pathlib.Path(__file__).parent.resolve()
+
+this_directory = pathlib.Path(__file__).parent
+long_description = (this_directory / "README.md").read_text()
+
 
 setup(
```

Comment:
Should now have an added dependency on lomas-core==0.3.3?

Reply:
Same for the client setup.py?

Reply:
Plus (that was my mistake in the pr where I made the server package installable), I think the install_requires list should contain all the server dependencies (more or less what is in the requirements file), otherwise it's only usable if someone manually installs the requirements.

Reply:
yes, I'll add them